.script/tests/KqlvalidationsTests/KqlValidationTests.cs

```diff
@@ -12,7 +13,7 @@ namespace Kqlvalidations.Tests
     public class KqlValidationTests
     {
         private readonly IKqlQueryAnalyzer _queryValidator;
-        private static readonly string DetectionPath = DetectionsYamlFilesTestData.GetDetectionPath();
+        private static readonly List<string> DetectionPath = DetectionsYamlFilesTestData.GetDetectionPath();
```

Comment:
change the names of the variable and the method to "DetactionPaths"

Reply:
Done